### DISCUSSION TITLE
Podman 5.x.x on Rocky Linux 8 and 9

### DISCUSSION THREAD

**Discussion by PetreHoffmann** [containers/podman#24529] [Q&A] [answered]
Hello Guys, thank you for your work.
Do you have in mind when are you releasing the package for Centos/RHEL Distros like Rocky Linux?
For now, only version 4.9.4 is available in AppStreams.

**Comment by rhatdan** [comment]
podman 5 should be available for Centos Stream very soon, as it is about to be released for RHEL.  We do not support packaging for other distros, but Rocky Linux will grab it from RHEL sources once it is released.

**Comment by baude** [comment, marked as answer]
its probably also worth mentioning that if you are doing a custom thing or even in general with derivatives, many of Podman's dependencies are gradually moving along too (run/crun or cni/netavaark).  So keep an eye on those too
